src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -240,6 +241,18 @@ internal string[] GetWindowsPowerShellCompatibilityModuleDenyList()
             return settingValue;
         }
 
+        internal List<string> GetWindowsPowerShellCompatibilityNoClobberModuleList()
```

Comment:
Can you make this method returns `string[]` like `GetWindowsPowerShellCompatibilityModuleDenyList`? It looks inconsistent to return different type when the methods roughly have the same functionalities.

Reply:
Updated to string array.